### ISSUE TITLE
[Beta] Conceptual error in the description of useMemo

### ISSUE THREAD

**Issue by kavinkuma6** [reactjs/reactjs.org#5212]
The marked sentence in the image of [this page](https://beta.reactjs.org/apis/react/useMemo) should be 

```
Memoization lets your component re-render only when dependencies are different.
```
<img width="1440" alt="Screenshot 2022-10-27 at 12 15 31 PM" src="https://user-images.githubusercontent.com/24503078/198210789-82f77076-c456-44bd-a882-1e7d78ffe74a.png">

**Comment by harish-sethuraman** [comment]
Hey thanks for sending PR.
wasn't this fixed by https://github.com/reactjs/reactjs.org/pull/5224
https://github.com/reactjs/reactjs.org/issues/5223?

**Comment by xnivaxhzne** [comment]
> Hey thanks for sending PR. wasn't this fixed by #5224 #5223?

@harish-sethuraman Hey, Yes correct. Also, the same error is in [this file](beta/src/content/apis/react/useCallback.md).
And I have updated my PR #5213 to fix that. 
Please have a look.

**Comment by kavinkuma6** [comment]
> Hey thanks for sending PR. wasn't this fixed by #5224 #5223?

@harish-sethuraman Hey, Yes correct. Also, the same error is in [this file](beta/src/content/apis/react/useCallback.md).
And I have updated my PR #5213 to fix that. 
Please have a look.
